src/main/java/decodex/parser/Parser.java

```diff
@@ -53,12 +55,13 @@ public String getCommandType(String userInput) throws ParserException {
     }
 
     /**
-     * Returns the Argument portion of the user input.
+     * Returns the Argument portion of the user input as an array.
+     * NOTE - Might be removed since DataCommand takes in only String rather than array.
      *
      * @param userInput The input specified by the user.
-     * @return The argument portion of the user input.
+     * @return The argument portion of the user input as an array.
      */
-    public String[] getUserArguments(String userInput) throws ParserException {
+    public String[] getUserArgumentsAsArray(String userInput) throws ParserException {

```

Comment:
As this is no longer used, perhaps discard this method? And also, I foresee we might use a third-party library or Regex for parsing line arguments so lets remove this method for now.